tests/ai/ai_test.py

```diff
@@ -0,0 +1,58 @@
+# Allows code to be run in root directory
+import sys
+from enum import Enum, auto
+
+from pydantic import BaseModel, Field
+
+from grimoire.ai.ai_provider import AIProvider, load_key
+
+sys.path[0] = sys.path[0].removesuffix("\\tests\\ai")
+
+
+from grimoire.core.generator.test import TestModule
+from grimoire.core.logger import LoggerSettings, LoggingLevel
+from grimoire.core.noise.global_seed import GlobalSeed
+from grimoire.story.names.name_generator import NamingSchema, NameGenerator
+
+
+class SettlementSize(Enum):
+    Small = auto()
+    Medium = auto()
+    Large = auto()
+
+
+class Citizen(BaseModel):
+    name: str
+    profession: str
+
+
+class SettlementData(BaseModel):
+    size: SettlementSize
+    name: str = Field(
+        description="The name of the settlement, which should end in town"
```

Comment:
Perhaps having them all end in town is a bad choice, I kind of would like the ai to have more freedom in naming. That being said, maybe we clarify that the names should be sort of old timey, or medieval?

Reply:
This was just me testing the efficacy of setting the description field with an arbitrary rule- this test file is certainly not going to be the actual technique used for generating the town